condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx

```diff
@@ -87,7 +87,7 @@ export function init(id: string, options: Options = {}) {
                 sortOrders.push(state.forms[i].position);
             }
 
-            if (sortOrderField instanceof HTMLInputElement) {
+            if (sortOrderField instanceof HTMLInputElement || sortOrderField instanceof HTMLTextAreaElement) {
```

Comment:
This field is always a hidden input so checking for ``HTMLTextAreaElement`` is not required here